From 27b498cecd284e36d31898b38df4ca276e4ce3f1 Mon Sep 17 00:00:00 2001
From: Aaron Smuts <asmuts@apache.org>
Date: Fri, 17 Nov 2006 15:03:10 +0000
Subject: [PATCH] Fixing stupid error in the thread pool manager.  If you
 specify the useBoundary value, it would alwasy be false.  If you didn't then
 it would be true!

JCS-14

git-svn-id: https://svn.apache.org/repos/asf/jakarta/jcs/trunk@476164 13f79535-47bb-0310-9956-ffa450edef68
---
 src/java/org/apache/jcs/utils/threadpool/ThreadPoolManager.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/java/org/apache/jcs/utils/threadpool/ThreadPoolManager.java b/src/java/org/apache/jcs/utils/threadpool/ThreadPoolManager.java
index 93ef8da7f..1b593ccff 100644
--- a/src/java/org/apache/jcs/utils/threadpool/ThreadPoolManager.java
+++ b/src/java/org/apache/jcs/utils/threadpool/ThreadPoolManager.java
@@ -355,7 +355,7 @@ protected PoolConfiguration loadConfig( String root )
         {
             try
             {
-                config.setUseBoundary( Boolean.getBoolean( (String) props.get( root + ".useBoundary" ) ) );
+                config.setUseBoundary( Boolean.valueOf( (String) props.get( root + ".useBoundary" ) ) );
             }
             catch ( NumberFormatException nfe )
             {
